src/styles.css

```diff
@@ -440,6 +466,20 @@ button {
   animation: hideSoundIcon 3s forwards;
 }
 
+.key-input {
```

Comment:
maybe create a wrapper to handle the absolute positioning and then flex for the child

Reply:
done